content/use-cases/use-an-escrow-as-a-smart-contract.md

```diff
@@ -0,0 +1,137 @@
+## Meet the prerequisites
+
+The party host (sender) must have:
+
+- An XRP Ledger [account](accounts.html#creating-accounts) that holds enough XRP to pay for escrow and any fees incurred.
+
+- Access to a secure signing environment, which includes having a network connection to a [`rippled` server](install-rippled.html) (any server) that they can submit signed transactions to. <!--#{ once set up secure signing tutorial is available, link to it from here }# -->
+
+The party band (receiver) must have:
+
+- An XRP Ledger [account](accounts.html#creating-accounts) that can receive the XRP paid by the escrow.
+
+- Access to a [`rippled` server](install-rippled.html) that they can use to look up the details of an XRP Ledger transaction hash and submit the fulfillment value to finish the escrow.
+
+The party planner (oracle) must have:
+
+- The ability to generate a condition and a fulfillment.
+
+- To be able to keep a secret (the fulfillment) until the time is right.
+
+- A way to communicate the fulfillment publicly or at least to the party band when the time is right.
+
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## Define the terms of the smart contract
+
+To create the escrow as a smart contract, the participants must first define the terms of the contract. In this scenario, the participants need to agree on the following details.
+
+- **Should the escrow disallow fulfillment until a specific time?**
+
+      While this is an option, the participants agree that it is unnecessary for their escrow. For conditionally-held escrows, enabling this option doesn't provide any additional security, since whether the escrow can be finished still depends entirely on whether the party planner (oracle) publishes the fulfillment before the expiration.
+
+- **Should the escrow expire?**
+
+      Absolutely yes. The participants agree that the escrow should expire after 12 noon the day after the party. This gives the party band (receiver) enough time to finish the escrow, after the party planner verifies that they fulfilled their end of the contract and publishes the cryptographic fulfillment. After expiration, the locked XRP returns to the party host's (sender's) account.
+
+      If the participants don't allow the escrow to expire and the party planner doesn't release the condition, the XRP stays locked in the escrow forever.
+
+- **How much XRP should the escrow lock up and potentially pay?**
+
+      The participants agree that the escrow should lock up and potentially pay 2000 XRP, which is the party band's fee.
+
+- **From which XRP Ledger account should the escrow lock up XRP for potential payment to the party band?**
+
+      The participants agree that the escrow should lock up and potentially pay XRP out of the party host's XRP Ledger account.
+
+- **Which XRP Ledger account should the escrow potentially pay XRP to?**
+
+      The participants agree that the escrow should potentially pay XRP to the party band's XRP Ledger account.
+
+
```

Comment:
Another (arguably the most important) part of the puzzle: how shall the oracle know that the receiver did in fact play at the party?

If the party planner trusts the band and the party host to tell the truth, the oracle could hear conflicting reports about whether the band even played. Part of the job of an oracle is to know the truth (that's where the name comes from), so the party planner should be at the party to see for themselves.

You could picture a case where the party planner has some trusted partygoer(s) who report whether the band played, but in that case the partygoers and the party planner are jointly acting the role of oracle.

Reply:
@mDuo13 -Yes, I see! In my mind, the party planner is at the party. Maybe I've seen too many movies with control-freak wedding planners in them so I just assumed the planner would be onsite. LOL. I will update the doc to make it clear that in this scenario, the party planner is at the party to witness the party band playing and therefore knows the truth firsthand.